aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/requantization.h

```diff
@@ -55,6 +55,31 @@ pytorch_qnnp_compute_scalar_requantization_params(
   return params;
 }
 
+static inline union pytorch_qnnp_fp32_requantization_params
+pytorch_qnnp_compute_scalar_fp32_requantization_params(
+    float scale,
+    uint8_t zero_point,
+    uint8_t min,
+    uint8_t max) {
+  /* Compute requantization parameters */
+  assert(scale < 1.0f);
+  assert(scale >= 0x1.0p-32f);
+
+  union pytorch_qnnp_fp32_requantization_params params;
+  params.scalar.scale = scale;
+  params.scalar.output_zero_point = zero_point;
+  params.scalar.output_max = max;
+  params.scalar.output_min = min;
+  params.scalar.min_less_zero_point = ((float)((int32_t)(uint32_t)min -
+      (int32_t)(uint32_t)zero_point));
+  params.scalar.max_less_zero_point = ((float)((int32_t)(uint32_t)max -
+      (int32_t)(uint32_t)zero_point));
+  params.scalar.magic = 12582912.0f;
```

Comment:
Could you add a comment here explaining what these magical variables do?

Reply:
I will copy paste the comment from elsewhere, where this is copied from. It is still little convoluted to wrap head around :).

Reply:
@supriyar, here is the comment that explains it: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qnnpack/src/requantization/fp32-neon.c#L97